project/web/gunicorn.conf

```diff
@@ -7,9 +7,8 @@ autorestart=true
 stopasgroup=true
 killasgroup=true
 stopwaitsecs=60
-stdout_logfile={{ log_dir }}/gunicorn.log
-redirect_stderr=true
-stderr_logfile={{ log_dir }}/gunicorn.error.log
+stdout_syslog=true
```

Comment:
Ack! You are probably right. It looks like you can set the log file to `syslog` and it will work in earlier versions (but still might need to upgrade): http://supervisord.org/logging.html

> One may set stdout_logfile or stderr_logfile to the special string “syslog”. In this case, logs will be routed to the syslog service instead of being saved to files.